spec/helper.rb

```diff
@@ -10,7 +10,43 @@
   Coveralls.wear!
 end
 
-unless system("nvim -nu NONE +q")
+module Support
+  def self.workspace
+    File.expand_path("../workspace", __FILE__)
+  end
+
+  def self.socket_path
+    file_path("nvim.sock")
+  end
+
+  def self.port
+    server = TCPServer.new("0.0.0.0", 0)
+
+    begin
+      server.addr[1]
+    ensure
+      server.close
+    end
+  end
+
+  def self.file_path(name)
+    File.join(workspace, name)
+  end
+
+  def self.setup_workspace
+    FileUtils.mkdir_p(workspace)
+  end
+
+  def self.teardown_workspace
+    FileUtils.rm_rf(workspace)
+  end
+
+  def self.child_argv(base: ["nvim", "-i", "NONE", "-u", "NONE", "-n"], extra: [])
```

Comment:
@jpleau I'm not using keyword syntax to stay compatible with 1.9 (as you saw from the test failure). Maybe this could just accept an optional array argument, i.e. `def self.child_argv(extra=[])`? I'm also fine with having no arguments, and making callers explicitly append to the array for simplicity, e.g. `Support.child_argv + ["+q"]`.